org.eclipse.winery.frontends/app/tosca-management/src/app/wineryLicenseModule/LicenseEngineApiData.ts

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+export interface Software {
+    id: string;
+    name: string;
+    url: string;
+    branch: string;
+    status: Software.StatusEnum;
+    files: number;
+    licensesEffective: Array<string>;
+    filesExcluded: number;
+    licensesAll: Array<string>;
+}
+
+export namespace Software {
```

Comment:
Why did you make this namespace here? I guess a simple [string enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) is the way to go here.

Reply:
changed